browser/src/Plugins/Api/LanguageClient/LanguageClientHelpers.ts

```diff
@@ -68,13 +68,14 @@ export const getTextFromContents = (contents: types.MarkedString | types.MarkedS
 
 export const bufferUpdateToTextDocumentItem = (args: Oni.BufferUpdateContext): types.TextDocumentItem => {
     const lines = args.bufferLines
-    const { bufferFullPath, filetype, version } = args.eventContext
+    const { bufferFullPath, filetype/* , modified  */} = args.eventContext
     const text = lines.join(os.EOL)
 
     return {
         uri: wrapPathInFileUri(bufferFullPath),
         languageId: filetype,
-        version,
+        // modified,
+        version: 0,
```

Comment:
Unfortunately this may cause problems. This is used in our client implementation of the "Language Server Protocol" (https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md).

The `TextDocumentItem` is used to notify the language server that the file has changed, and it should update its in-memory representation. This is important for stuff like autocomplete, enhanced syntax highlighting, etc.

Some implementations rely on the `version` being incremented, otherwise they won't bother re-parsing or doing work to update that in-memory version... this would then come up as issues during completion.